### DISCUSSION TITLE
Asking for instructions for clarity

### DISCUSSION THREAD

**Discussion by UnrealisticDino** [canonical/steam-snap#207] [Q&A] [answered]
What logs should I be including in posts and where are they saved by default? Are there launch options I should be using to generate logs? If a game works what logs would be useful to include? I haven't been able to figure out how to label my posts.

**Comment by ashuntu** [comment, marked as answer]
You won't be able to label posts yourself, but if you have a label in mind, you can write it in your post for one of the maintainers to label it for you.

For logs, the easiest way is to run Steam from a terminal with `snap run steam` and then you will be able to see and copy any logs you need. See the wiki for more info: https://github.com/canonical/steam-snap/wiki/Troubleshooting#view-logs. It's best if you only copy logs that are generated *after* you start a game if it's a game specific issue. Its also useful to include what happens when you run a game, like if the game doesn't even start, or crashes at a certain point, or has certain issues, etc.

If a game works perfectly you do not need to include any logs, only if there are issues.

**Reply by UnrealisticDino** [reply]
Thank you for the reply. More games work with the edge version compared to the stable version and as of yesterday ubuntu 22.04 has nvidia 530 drivers but 23.04 only has up to 525. What would be the most useful configuration?

**Reply by ashuntu** [reply]
It really just depends on what works best for your setup. For example, on my laptop anything past 515 doesn't work well for me, but I know others are able to use the newer driver versions just fine. It may just take some trial and error. Newer driver versions aren't going to be super necessary unless you are running a brand new game or running into issues with older versions
